infrastructure/main.tf

```diff
@@ -197,3 +210,35 @@ data "azurerm_key_vault_secret" "encryption_public_key" {
   name         = "encryption-public-key"
   key_vault_id = "${module.send-letter-key-vault.key_vault_id}"
 }
+
+# region staging DB secrets
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_USER" {
+  key_vault_id = "${module.send-letter-key-vault.key_vault_id}"
+  name         = "${var.component}-STAGING-POSTGRES-USER"
```

Comment:
would be nice to use same naming convention as we have used in other repos for staging

Reply:
do you mean using lower case?

Reply:
` name         = "${var.component}-staging-db-user"`

Reply:
I used the existing format

Reply:
i was thinking about making a comment but reasoned myself not to